guidelines/GOLANG.md

```diff
@@ -0,0 +1,31 @@
+Golang guidelines
+=====================
+
+These are the Digital Publishing guidelines for developing code using Golang. They can also be used as a code review
+checklist.
+
+#### General
+* Start with the language level recommendations for Golang in [Effective Go](https://golang.org/doc/effective_go.html) 
+and the [code review guidelines](https://github.com/golang/go/wiki/CodeReviewComments)
+* [Listen for OS signals](https://golang.org/pkg/os/signal/) and gracefully shutdown the app when one is received
+
+#### Tooling
+
+* Use [gometalinter](https://github.com/alecthomas/gometalinter) to identify code issues
+* Prefer using [GoConvey](https://github.com/smartystreets/goconvey) as a test tool
+* Use [govendor](https://github.com/kardianos/govendor) for vendoring dependencies
```

Comment:
There has been no discussions. I was just gathering the current tooling at this stage. It certainly seems like `dep` is worth looking at if its likely to become standard anyway.